### PR TITLE
fix: using let instead of var and all var declarations at top

### LINKED ISSUE #7941: (JS-0102) Use let instead of var & Prefer var declarations be placed at the top of their scope
## Description
Declare variables at the top of their scope as it improves code readability, performance and also helps in code navigation.

## Occurrences
There are 2 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/appsmithorg/appsmith/issue/JS-0102/occurrences/](https://deepsource.io/gh/appsmithorg/appsmith/issue/JS-0102/occurrences/)

### PR DESCRIPTION
**Pull request by samyakjain10**
## Description

Fixes #7941 

## Type of change

> Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Comments/AddComments_spec.js [hunk 1]
```diff
@@ -22,9 +22,10 @@ function setFlagForTour() {
 function typeIntoDraftEditor(selector, text) {
   cy.get(selector).then((input) => {
     var textarea = input.get(0);
+    var textEvent = document.createEvent("TextEvent");
```
Comment (Rishabh-Rathod) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Comments/AddComments_spec.js:25]: All the variables, declared using `var` are expected to be converted to `let` or `const`, please do change in mentioned lines also.

**Review by Rishabh-Rathod — CHANGES_REQUESTED** [review]

**Comment by samyakjain10** [comment]
I have made the required changes. Please review them

**Review by Rishabh-Rathod — CHANGES_REQUESTED** [review]
@samyakjain10 Seems like there is one more `var`  occurrence as per the issue in https://github.com/appsmithorg/appsmith/blob/release/app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/Comments/utils.js#L6-L6

**Review by Rishabh-Rathod — APPROVED** [review]
Seems like this was due to merge with release branch, Hence approving your PR

**Pull request merged**